packages/component-library/stories/button.markdown.md

```diff
@@ -0,0 +1,36 @@
+# Button Component
+
```

Comment:
Per our in person discussion, recommend continuing to explore options here that meet the following criteria:

- Valuable to design system users
- Simple to keep in sync with current state of code
- Future compatibility / adaptation to Storybook docs

Reply:
All of the above requested changes have been made.